src/constants.tsx

```diff
@@ -109,3 +112,339 @@ export const objectCoreResource = "0x1::object::ObjectCore";
+    chainId: 1,
+    tokenAddress: null,
+    faAddress:
+      "0xd39fcd33aedfd436a1bbb576a48d5c7c0ac317c9a9bb7d53ae9ffb41e8cb9fd9",
+    name: "Find Out Points",
+    symbol: "FFO",
+    decimals: 8,
+    panoraSymbol: null,
+    bridge: null,
+    logoUrl: "",
+    websiteUrl: "https://www.letsfindout.ai/",
+    category: "Native",
+    isInPanoraTokenList: false,
+    isBanned: false,
+    panoraOrderIndex: 999999999,
+    coinGeckoId: null,
+    coinMarketCapId: null,
+  },
+  "0x357b0b74bc833e95a115ad22604854d6b0fca151cecd94111770e5d6ffc9dc2b": {
+    chainId: 1,
+    tokenAddress: null,
+    faAddress:
+      "0x357b0b74bc833e95a115ad22604854d6b0fca151cecd94111770e5d6ffc9dc2b",
+    name: "Tether USD",
+    symbol: "USDt", // Turns out USDt is the symbol when unicode USD₮ is not supported
+    decimals: 6,
+    bridge: null,
+    panoraSymbol: null,
+    logoUrl: "https://tether.to/images/logoCircle.svg",
+    websiteUrl: "https://tether.to",
+    category: "Native",
+    isInPanoraTokenList: true,
+    isBanned: false,
+    panoraOrderIndex: 2,
+    coinGeckoId: "tether",
+    coinMarketCapId: 825,
+    native: true,
+  },
+  "0x50038be55be5b964cfa32cf128b5cf05f123959f286b4cc02b86cafd48945f89": {
+    chainId: 1,
+    tokenAddress: null,
+    faAddress:
+      "0x50038be55be5b964cfa32cf128b5cf05f123959f286b4cc02b86cafd48945f89",
+    name: "BlackRock BUIDL",
+    symbol: "BUIDL",
+    decimals: 6,
+    panoraSymbol: null,
+    bridge: null,
+    logoUrl:
+      "https://securitize-public-files.s3.us-east-2.amazonaws.com/ClientDocuments/buidl/token-logo-256.png",
+    websiteUrl: "http://securitize.io/blackrock/BUIDL",
+    category: "Native",
+    isInPanoraTokenList: true,
+    isBanned: false,
+    panoraOrderIndex: 3,
+    coinGeckoId: "blackrock-usd-institutional-digital-liquidity-fund",
+    coinMarketCapId: null,
+  },
+};
+
+/**
+ * Coin supply limit overrides
+ *
+ * These must be vetted to not have mint or burn capabilities
+ */
+export const supplyLimitOverrides: Record<string, bigint> = {
+  "0x268d4a7a2ad93274edf6116f9f20ad8455223a7ab5fc73154f687e7dbc3e3ec6::LOON::LOON":
+    1000000000000000n,
+};
+
+export const EMOJICOIN_REGISTRY_ADDRESS =
+  "0x4b947ed016c64bde81972d69ea7d356de670d57fd2608b129f4d94ac0d0ee61";
+
+export const nativeTokens: Record<string, string> = {
+  "0x1::aptos_coin::AptosCoin": "APT",
+  "0xa": "APT",
+  "0xA": "APT",
+};
+export const labsVerifiedTokens: Record<string, string> = {
+  "0x357b0b74bc833e95a115ad22604854d6b0fca151cecd94111770e5d6ffc9dc2b": "USDt",
+  "0xbae207659db88bea0cbead6da0ed00aac12edcdda169e591cd41c94180b46f3b": "USDC",
+  "0xd39fcd33aedfd436a1bbb576a48d5c7c0ac317c9a9bb7d53ae9ffb41e8cb9fd9":
+    "Find Out Points",
+  "0x50038be55be5b964cfa32cf128b5cf05f123959f286b4cc02b86cafd48945f89": "BUIDL",
+};
+export const MARKED_AS_SCAM = "Marked as scam";
+export const MARKED_AS_POSSIBLE_SCAM = "Marked as possible scam";
+export const labsBannedTokens: Record<string, string> = {
```

Comment:
Should have left the comment here:

"0xf658475dc67a4d48295dbcea6de1dc3c9af64c1c80d4161284df369be941dafb::moon_coin::MoonCoin",
      "0xbbc4a9af0e7fa8885bda5db08028e7b882f2c2bba1e0fedbad1d8316f73f8b2f::Questss::Quests",
      "0xbbc4a9af0e7fa8885bda5db08028e7b882f2c2bba1e0fedbad1d8316f73f8b2f::quest::Quest",
      "0xbc106d0fef7e5ce159423a1a9312e011bca7fb57f961146a2f88003a779b25c2::QUEST::QUEST",
      "0xbbc4a9af0e7fa8885bda5db08028e7b882f2c2bba1e0fedbad1d8316f73f8b2f::Questseess::Questsee",
      "0xbbc4a9af0e7fa8885bda5db08028e7b882f2c2bba1e0fedbad1d8316f73f8b2f::graffio::GRAFFIO",
      "0xbbc4a9af0e7fa8885bda5db08028e7b882f2c2bba1e0fedbad1d8316f73f8b2f::ograffioo::Ograffioo",
      "0xbbc4a9af0e7fa8885bda5db08028e7b882f2c2bba1e0fedbad1d8316f73f8b2f::ograffio::Ograffio",
      "0xbbc4a9af0e7fa8885bda5db08028e7b882f2c2bba1e0fedbad1d8316f73f8b2f::memes::Memecoins",
      "0xbbc4a9af0e7fa8885bda5db08028e7b882f2c2bba1e0fedbad1d8316f73f8b2f::meme::Memecoin",
      "0xbbc4a9af0e7fa8885bda5db08028e7b882f2c2bba1e0fedbad1d8316f73f8b2f::tortugafin::Tortugafin",
      "0x397071c01929cc6672a17f130bd62b1bce224309029837ce4f18214cc83ce2a7::USDC::USDC",
      "0xbbc4a9af0e7fa8885bda5db08028e7b882f2c2bba1e0fedbad1d8316f73f8b2f::Ogrf::Ogrff",
      "0x24a808cbdeee2e8b771fec7ce9140bab2cbbf89426bf135d6b5d88e6736bb038::EZMoney::EZ_Money",
      "0x48327a479bf5c5d2e36d5e9846362cff2d99e0e27ff92859fc247893fded3fbd::APTOS::APTOS",
      "0xbbc4a9af0e7fa8885bda5db08028e7b882f2c2bba1e0fedbad1d8316f73f8b2f::Memecoins::Aptmeme",
      "0xbde343d6d2a7b407b1314c6c6ef771ebd72b95f9f67b688bbaea5ece4b98bb57::maincoin::Aptmeme",
      "0xbe5e8fa9dd45e010cadba1992409a0fc488ca81f386d636ba38d12641ef91136::maincoin::Aptmeme",
      "0xe1d39a72bd69bc2ebfe008bb925badb23a32883b077218b9e167f74cf703db1a::uptos::UptosCoin",
      "0x50788befc1107c0cc4473848a92e5c783c635866ce3c98de71d2eeb7d2a34f85::aptos_coin::AptosCoin",
      "0x50788befc1107c0cc4473848a92e5c783c635866ce3c98de71d2eeb7d2a34f85::apt_rewards::APTRewards",
      "0x50788befc1107c0cc4473848a92e5c783c635866ce3c98de71d2eeb7d2a34f85::aptos_coin::AptosCoin",
      "0xb8aed93b62950ba62deacf5c5faca89922f87972e11fc594cdc2ae1d0a5d02de::CHILLGUY::CHILLGUY",
      "0xabf27cc0cba747b355f4ed64e9263ddf1e6b38d2caf570df5b5777f722fa861e::BULLY::BULLY",